bitnami/spring-cloud-dataflow/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: spring-cloud-dataflow
-version: 0.2.9
+version: 0.2.10
```

As there are different additions and not just bug fixing, I would bump the chart version in a minor instead of a patch
```suggestion
version: 0.3.0
```